stylesheets/components/ratings.css

```diff
@@ -0,0 +1,49 @@
+.tr-ratings-wrapper .fa-star{
+    color: var(--tr-warning);
+    /* font-size: var(--fs-sm); */
```

Comment:
remove comment code

Reply:
Thank you for pointing it out, I have fixed it in the latest commit